scripts/content.js

```diff
@@ -1,34 +1,29 @@
-const hideNews = (page, response) => {
-  console.log('loaded!');
+let currentPage;
+let currentResponse;
 
-  if (response && page === 'feed') {
-    let newsFeedModule = document.querySelector('#feed-news-module');
+const hideNews = () => {
+  console.log('loaded!');
+  console.log({ currentResponse, currentPage });
+  if (currentResponse && currentPage === 'feed') {
+    let newsFeedModule = document.getElementById('feed-news-module');
     console.log(newsFeedModule, 'news feed module?');
     newsFeedModule.classList.add('hidden');
-  } else if (response && page === 'news') {
```

Comment:
I changed these variable names so that the variables would be available in the global scope but might revert back to my original changes, with the variables passed to `hideNews` as arguments.